src/zap_hooks/helpers/utilities.py

```diff
@@ -0,0 +1,44 @@
+from sys import exit
+from typing import Dict, Iterable, NoReturn
+
+import src.zap_hooks.helpers.constants as Constants
+from src.zap_hooks.model.log_level import LogLevel, loggerFunc
+
+
+def log(message: str, log_level: LogLevel = LogLevel.INFO) -> None:
+    logFunc = loggerFunc.get(log_level)
+    logFunc(str(message))
+
+
+def print_line_separator() -> None:
+    print(
+        "----------------------------------------------------------------------------------------------------------"
```

Comment:
can you use the logging.py function for this?

Reply:
removed, wasnt being used